spec/models/post_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it 'Testing for no Title' do
+    post = Post.create
+    expect(post).to_not be_valid
+  end
+
+  it 'Testing post CommentCounter number negative' do
+    post = Post.create(AuthorId: 1, Title: 'Tanusri', Text: 'anything')
+    post.CommentCounter = -10
+    expect(post).to_not be_valid
+  end
+
+  it 'Testing post LikeCounter number negative' do
+    post = Post.create(AuthorId: 1, Title: 'Tanusri', Text: 'anything')
+    post.LikeCounter = -10
+    expect(post).to_not be_valid
+  end
+
+  it 'Title should be present' do
+    post = Post.create(AuthorId: 1, Title: 'Tanusri', Text: 'anything')
+    post.Title = nil
+    expect(post).to_not be_valid
+  end
+
+  it 'Title should not exceed 250 characters' do
+    subject.Title = 'a' * 256
+    expect(subject).to_not be_valid
+  end
+
+  it 'Has a latest_comments method' do
+    post = Post.new(Title: 'one post', CommentCounter: 5, LikeCounter: 8)
+    expect(post).to respond_to(:latest_comments)
+  end
```

Comment:
- [ ] As the previous reviewer mentioned kindly make sure to fix these errors so the rspec tests are passing 👇🏼 
![Screenshot_20230128_204700](https://user-images.githubusercontent.com/25227022/215287931-b297c047-f49f-4207-a5ac-7f4ddab7a0ae.png)